datm/cime_config/config_component.xml

```diff
@@ -10,9 +10,10 @@
        This file may have atm desc entries.
   -->
   <description modifier_mode="1">
-    <desc atm="DATM[%QIA][%WISOQIA][%CRU][%CRUv7][%GSWP3v1][%MOSARTTEST][%NLDAS2][%CPLHIST][%1PT][%NYF][%IAF][%JRA][%JRA-1p4-2018][%JRA-1p5-2023][%JRA-RYF8485][%JRA-RYF9091][%JRA-RYF0304][%SIMPLE]"> Data driven ATM </desc>
+    <desc atm="DATM[%QIA][%WISOQIA][%CRUJRA2024][%CRU][%CRUv7][%GSWP3v1][%MOSARTTEST][%NLDAS2][%CPLHIST][%1PT][%NYF][%IAF][%JRA][%JRA-1p4-2018][%JRA-1p5-2023][%JRA-RYF8485][%JRA-RYF9091][%JRA-RYF0304][%SIMPLE]"> Data driven ATM </desc>
     <desc option="QIA"> QIAN data set </desc>
     <desc option="WISOQIA">QIAN with water isotopes</desc>
+    <desc option="CRUJRA2024"> CRUJRA 2024 data set </desc>
```

Comment:
From meeting with @ekluzek 
See if I should remove %CRU

Reply:
@ekluzek I'm just adding a reminder
You will follow up with me about the above (whether to remove %CRU).

Reply:
Yes, let's remove plain %CRU. I made this an issue #324

Reply:
#323

Reply:
I removed plain CRU; however, in the same file I see that CRUv7 does NOT appear in the lists of defaults that CRUJRA appears in. Should it?

Reply:
Unless I hear otherwise, I will assume that CRUv7 is fine since it has worked, as far as I know, correctly all along.

Reply:
v7 is there, just two lines below on line 17...

Reply:
As an option, yes, but not further down where various defaults are set.

Reply:
This test
`ERI_D_Ld9_P48x1.f10_f10_mg37.I2000Clm50BgcCru.izumi_nag.clm-reduceOutput.C.0212-132154iz_nag`
answers the question I raised in this thread. I need to add CRUv7 to the various defaults to prevent this (and similar tests) from giving DIFFs from baseline.

Reply:
Faster turnaround on derecho shows that "Cru" tests do not give DIFFs from baseline now, so I will resolve this conversation.